### PR TITLE
AOM-87 :  Display appropriate message when start all button is clicked

### PR DESCRIPTION
**Pull request by justmesam**
## JIRA TICKET NAME:  
[AOM-87 :  Display appropriate message when start all button is clicked](https://issues.openmrs.org/browse/AOM-87)

#### Currently :
When you click start all modules while all modules are running it displays a misleading message that the modules have been started though they were already running.

#### After fix : 
When you click on the start all modules button it gives you a message that they are already started.

### CODE REVIEW

**Comment by Charpell** [comment]
Looks good to me.
Great job :+1:

**Comment by mkiterian** [comment]
Looks good to me

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -161,6 +162,27 @@ export default class ManageApps extends React.Component {
     });
   }
 
+  handleStartModules(){
+    const { appList } = this.state;
+    const allStartedModules = [];
+    appList.forEach((addon) => {
```
Comment (joel-ace) [inline app/js/components/manageApps/ManageApps.jsx]: Nice work. You can also achieve the same thing using `const allStartedModules = appList.filter(addon => addon.appDetails.started)`

File: app/js/components/manageApps/ManageApps.jsx [hunk 2]
```diff
@@ -630,12 +652,12 @@ export default class ManageApps extends React.Component {
               <h2 className="manage-addon-title">Add-on Manager</h2>
               <span className="pull-right manage-settings-wrapper">
                 <span id="startall-modules-btn"
-                  className="btn btn-secondary"
```
Comment (LWanjiru) [inline app/js/components/manageApps/ManageApps.jsx:633]: I recommend that you only make changes that directly affect the ticket directly, ie. formatting changes should be done separately. They make the PR a little noisy

File: app/js/components/manageApps/ManageApps.jsx (hunk 1 above)
Comment (justmesam) [reply app/js/components/manageApps/ManageApps.jsx]: Cool let me update that 👍

File: app/js/components/manageApps/ManageApps.jsx (hunk 2 above)
Comment (justmesam) [reply app/js/components/manageApps/ManageApps.jsx:633]: will take care of that next time. thanks

**Comment by justmesam** [comment]
@dkayiwa I have responded to all reviews

**Pull request merged**